docs/overview/basic_concurrency.md

```diff
@@ -141,31 +156,35 @@ The following table summarizes some of the sequential operations and their corre
 | Reduces many values            | `ZIO.reduceAll`   | `ZIO.reduceAllPar`   |
 | Merges many values             | `ZIO.mergeAll`    | `ZIO.mergeAllPar`    |
 
-For all parallel operations, if one effect fails, others will be interrupted to minimize unnecessary computation.
+Because all these parallel operators return all the results, if any effect being parallelized fails, ZIO will automatically cancel the other running effects, because their results will not be used.
```

Comment:
```suggestion
Because all these parallel operators return all the results, if any effect being parallelized fails, ZIO will automatically cancel the other running effects, as their results will not be used.
```